ossl/src/cipher.rs

```diff
@@ -285,8 +288,8 @@ impl OsslCipher {
                 ctx.aead_setup(alg, &aead)?
             }
             EncAlg::AesCts(_, mode) => {
-                params = OsslParam::with_capacity(1);
-                params.add_const_c_string(
+                let mut builder = OsslParamBuilder::with_capacity(1);
```

Comment:
this should be called params_builder

Reply:
I confused myself.  When I first read your comment, I somehow assumed you wanted every instance of `OsslParamBuilder` to be called `params_builder`.  But now I saw that the change you commented on indeed stands out for renaming `params` to `builder`.  I think this was the first call site I converted, and I decided not to rename the variable anywhere else.

I see two options now:
- rename the variable here to `params`
- rename all instances of `OsslParamBuilder` to `params_builder`

What do you prefer?